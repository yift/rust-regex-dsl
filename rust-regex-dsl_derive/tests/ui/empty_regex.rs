pub use rust_regex_dsl::regex;

fn main() {
    let regex = regex!("");
    println!("{}", regex);
}