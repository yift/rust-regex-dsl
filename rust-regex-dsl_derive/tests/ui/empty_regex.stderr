error: Empty regex is not supported
 --> tests/ui/empty_regex.rs:4:24
  |
4 |     let regex = regex!("");
  |                        ^^
